README.md

```diff
@@ -240,6 +241,17 @@ const mapDispatchToProps = (dispatch, props) => {
 };
 ```
 
+The result of the promise returned by `mutateAsync` will be the following object:
+
+| Name | Type | Description |
+|:----|:-----|:-----|
+| status | number | HTTP status code
+| body | object | Response body auto-parsed by SuperAgent
+| text | string | Unparsed response body string
+| duration | number | Elapsed time between the action dispatched and resolved
```

Comment:
I'd prefer to keep it high level. Something like "The total duration from the start of the query to receiving the full response."